### ISSUE TITLE
Compile error: cannot convert '<brace-enclosed initializer list>' to 'llvm::ElementCount'

### ISSUE THREAD

**Issue by guoyunhe** [RPCS3/rpcs3#13193]
### Quick summary

I am trying to compile rpcs3 with system LLVM 13. Get this error:

```
/home/abuild/rpmbuild/BUILD/rpcs3-0.0.26~git20230107/rpcs3/Emu/Cell/PPUTranslator.cpp:513:48: error: cannot convert '<brace-enclosed initializer list>' to 'llvm::ElementCount'
  513 |                 return ConstantVector::getSplat({count, false}, cv);
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/llvm/IR/ConstantFolder.h:20,
                 from /usr/include/llvm/IR/IRBuilder.h:25,
                 from /home/abuild/rpmbuild/BUILD/rpcs3-0.0.26~git20230107/rpcs3/Emu/CPU/CPUTranslator.h:19:
```

### Attach a log file

-

**Comment by oltolm** [comment]
This should be fixed in #13500.

**Comment by guoyunhe** [comment]
Yes, it can be built successfully now.
